liveblog/liveblog
# [LBSD-2878] - Add "no-cache" for poll requests on embed

### Purpose

This PR adds "no-cache" to the Cache control attribute for poll requests so that they re-validate with the server to avoid the use of stale data. 

### What has changed

- Added "no-cache" attribute to poll requests on the embed

### Steps to test

1. Open the Editor and publish a new poll.
2. Open the preview and place a vote.
3. Observe that the request to fetch a poll first to get the etag before patching has the "no-cache" attribute in the Cache Control option in the request headers

Resolves: [LBSD-2878](https://sofab.atlassian.net/browse/LBSD-2878)

[LBSD-2878]: https://sofab.atlassian.net/browse/LBSD-2878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
